src/core/public/chrome/chrome_service.tsx

```diff
@@ -181,6 +196,17 @@ export class ChromeService {
       localStorage.setItem(IS_LOCKED_KEY, `${isLocked}`);
     };
 
+    const collapsibleNavHeaderRender = this.collapsibleNavHeaderRender
+      ? () =>
+          this.collapsibleNavHeaderRender
```

Comment:
Duplicate judgement on `this.collapsibleNavHeaderRender`, maybe we only need one?

Reply:
Good point, updated now